src/lib/features/feature-toggle/tests/feature-toggle-service.limit.test.ts

```diff
@@ -97,7 +97,17 @@ describe('Strategy limits', () => {
         const { featureToggleService, featureStrategiesStore } =
             createFakeFeatureToggleService({
                 getLogger,
-                flagResolver: alwaysOnFlagResolver,
+                flagResolver: {
```

Comment:
The reason these tests broke needs to be investigated, I suspect the tests might rely on lacking implementations in fake methods

Reply:
Ok, so they're failing because the environment property is undefined. It's not marked as optional in `optionallyDisableFeature` though, so I'm a bit unsure what the right direction here is. Fixing the tests and always assuming the environment parameter is set, or escaping early if it is not set